Engine/ac/global_debug.cpp

```diff
@@ -119,7 +119,9 @@ void script_debug(int cmdd,int dataa) {
         delete tempw;
         delete view_bmp;
         gfxDriver->DestroyDDB(ddb);
-        wait_until_keypress();
+        // Wait until key press
+        while (!ags_kbhit());
+        ags_getch();
```

Comment:
I have only one question, why did you remove "wait_until_keypress"? I used that as a convenience helper function (to avoid putting "while" construct in place every time.

Reply:
It actually didn't work well with some graphics modes in SDL because we were no longer rendering to screen.  In my SDL version I refactored code that used wait_for_keypress to have a render & check for key event loop.